locals.tf

```diff
@@ -32,8 +32,9 @@ locals {
   }
 
   lets_encrypt_dns_challenged_domains = {
-    "trusted.ci.jenkins.io" = "service_principal"
-    ## TODO: add support for workload identities:
-    # "cert.ci.jenkins.io" = "managed_identity"
+    "trusted.ci.jenkins.io" = "2024-04-03T20:00:00Z"
+    "cert.ci.jenkins.io"    = "2024-04-03T21:00:00Z"
+    ## TODO: add support for workload identities by NOT providing any expiration date
```

Comment:
```suggestion
    # TODO: add support for workload identities by providing an empty expiration date
```
nit: Easier to understand IMO

Reply:
Should another domain be cited as example?

Reply:
> Should another domain be cited as example?

No advise: it's only a "TODO + example" for later. Ideally, cert.ci is the next likely candidate hence why it is used.
I don't expect this comment to be a "uncomment and deploy": if the person doing this see their build check red, that will teach them :D